store-api-server/script/migrate

```diff
@@ -48,6 +48,7 @@ CREATE INDEX IF NOT EXISTS paypoint_entry_default_uuid_idx ON paypoint."entry.de
 CREATE INDEX IF NOT EXISTS peppermint_command_name_state_idx ON peppermint.operations((command->>'name'), state);
 CREATE INDEX IF NOT EXISTS peppermint_command_token_id_state_idx ON peppermint.operations(state, (command->>'name'), ((command->'args'->'token_id')::int));
 CREATE INDEX IF NOT EXISTS peppermint_token_id_handler_to_idx ON peppermint.operations(((command->'args'->'token_id')::int), (command->>'handler'), (command->'args'->>'to_address'));
+CREATE INDEX IF NOT EXISTS peppermint_handler_idx ON peppermint.operations((command->'handler'));
```

Comment:
has to be the `->>` operator. i noticed over the weekend that when defining these json expression indexes it's important you use the exact same expression as is used in the query, otherwise postgres wont use the index.